js/src/components/SchemaTab.js

```diff
@@ -0,0 +1,234 @@
+import React, { Fragment } from "react";
+import styled from "styled-components";
+import { Select } from "@yoast/components/src/select/Select";
+import { __, sprintf } from "@wordpress/i18n";
+import { createPortal } from "react-dom";
+import HelpIcon from "@yoast/components/src/help-icon/HelpIcon";
+import SidebarCollapsible from "./SidebarCollapsible";
+import PropTypes from "prop-types";
+import interpolateComponents from "interpolate-components";
+
+const SchemaContainer = styled.div`
+	padding: 16px;
+`;
+
+const schemaPageOptions = [
+	{
+		name: __( "Web Page", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Item Page", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "About Page", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "FAQ Page", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "QA Page", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Profile Page", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Contact Page", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Medical Web Page", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		name: __( "Collection Page", "wordpress-seo" ),
+		value: "option-9",
+	},
+	{
+		name: __( "Checkout Page", "wordpress-seo" ),
+		value: "option-10",
+	},
+	{
+		name: __( "Real Estate Listing", "wordpress-seo" ),
+		value: "option-11",
+	},
+	{
+		name: __( "Search Results Page", "wordpress-seo" ),
+		value: "option-12",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
+		value: "option-13",
+	},
+];
+
+const schemaArticleOptions = [
+	{
+		name: __( "Article", "wordpress-seo" ),
+		value: "option-1",
+	},
+	{
+		name: __( "Social Media Posting", "wordpress-seo" ),
+		value: "option-2",
+	},
+	{
+		name: __( "News Article", "wordpress-seo" ),
+		value: "option-3",
+	},
+	{
+		name: __( "Advertiser Content Article", "wordpress-seo" ),
+		value: "option-4",
+	},
+	{
+		name: __( "Satirical Article", "wordpress-seo" ),
+		value: "option-5",
+	},
+	{
+		name: __( "Scholary Article", "wordpress-seo" ),
+		value: "option-6",
+	},
+	{
+		name: __( "Tech Article", "wordpress-seo" ),
+		value: "option-7",
+	},
+	{
+		name: __( "Report", "wordpress-seo" ),
+		value: "option-8",
+	},
+	{
+		/* translators: %1$s expands to "- " (a hyphen and a space), %2$s expands to " -" (a space and a hyphen) */
+		name: sprintf( __( "%1$sNone%2$s", "wordpress-seo" ), "- ", " -" ),
+		value: "option-9",
+	},
+];
+
+/* translators: %1$s and %2$s expand to a link to the Search Appearance Settings page */
+const footerText = sprintf(
+	__( "You can change the default type for Posts in your %1$sSearch Appearance Settings%2$s.", "wordpress-seo" ),
```

Comment:
This now Always reads `Posts`, even on Pages.
Perhaps, in an earlier iteration, the metabox on Pages did not show this footer?

Make sure this either reads `Pages` on Pages, or hide the footer if that is the spec.